README.adoc

```diff
@@ -84,30 +84,37 @@ mobile-security-service-operator in your OpenShift cluster as follows:
 ....
 make cluster/prepare
 make install
+oc set env deployment/$(oc get -f deploy/operator.yaml --template "{{.metadata.name}}") OPENSHIFT_HOST="${OPENSHIFT_HOST}"
 ....
 
 
-== TODO
+== Post installation work
 
-Please take the following steps manually for now after the above install steps
+TODO
 
 ```
```

Comment:
Following an example to add a code in adoc.


[source,yaml]
----
$ make setup
----

Reply:
What Ali has here is also fine. If you click the "rich diff" button on the top right of this file here in the review, you can see how GitHub will render it:

![image](https://user-images.githubusercontent.com/442386/60712692-7c491180-9f0f-11e9-958e-607fded25589.png)